src/de/willuhn/jameica/hbci/gui/controller/LicenseControl.java

```diff
@@ -99,6 +98,7 @@ public Part getLibList()
       }
       catch (Exception e)
       {
+        // TODO hier auch "info" statt "infos[0]"?
         Logger.error("unable to parse " + infos[0],e);
```

Comment:
Hier ist mir die Logik nicht ganz klar. Deswegen der `TODO`-Marker. Was soll hier in der Log-Nachricht ausgegeben werden? `info[0]` oder `info[i]`?

Reply:
Wegen dieser Frage ist der Pullrequest noch im Status "draft".